bin/notificators/hard_threshold_notificator.py

```diff
@@ -33,29 +36,69 @@ def handle_single_event(self, body):
         if is_send_to_user or is_send_to_admin_channel:
             self.send_slack_notification(query, is_send_to_user, is_send_to_admin_channel)
 
+    def get_slack_user_id(self, username):
```

Comment:
If I understand correctly we will now get user_id every run, not based on mapping, right?
It might be risky (quota limit), but on the other hand, we rarely see more than 10 notifications in one minute. I think it can stay. We can get rid of user mappings, though

Reply:
I plan on getting rid of user mappings as soon as we fully switch to SSO. For now, it looks like many people still use regular IAM accounts.